src/main/java/com/google/codeu/data/Message.java

```diff
@@ -29,15 +29,21 @@
   private long timestamp;
 
   /**
-   * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
+   * Constructs a new {@link Message} posted by {@code user} with {@code text} content to {@code recipient}. Generates a
```

Comment:
I think if you run this through the google-code-formatter code that I linked to, that will fix the wrapping issue that you're seeing.